### ISSUE TITLE
[sphinx] m2r2 extension prohibits usage of multiple jobs

### ISSUE THREAD

**Issue by marscher** [BAMWelDX/weldx#508]
If I comment out this extension I can use multiple jobs during the Sphinx build.
I will update this issue, when I know more.

**Comment by CagtayFabry** [comment]
we only use this for the readme and changelog right now, correct?
Moving these over to RST would be worth it for the parallel build support imo

**Comment by marscher** [comment]
That is correct. I agree that we should consider switching now although RST is more verbose and has more hiccups with block idention (new-lines) etc.

**Comment by marscher** [comment]
It looks like sphinx_asdf is causing the same PicklingError. Eventually both extensions use a legacy API of Sphinx to get things done.

**Comment by CagtayFabry** [comment]
that's too bad :/
we could probably fix this down the in the `sphinx-weldx` branch but it sounds like quite some work ...
